Dockerfile

```diff
@@ -15,20 +16,13 @@ RUN echo -e "[WandiscoSVN]" \
  "\nbaseurl=http://opensource.wandisco.com/centos/6/svn-1.9/RPMS/$basearch/" \
  "\nenabled=1" \
  "\ngpgcheck=0" > /etc/yum.repos.d/wandisco.repo
-RUN microdnf -y install \
-  java-11-openjdk-headless \
-  openssh-clients \
-  unzip \
-  wget \
-  git \
-  subversion \
-  maven \
- && microdnf -y clean all
-ARG WINDUP=https://repo1.maven.org/maven2/org/jboss/windup/tackle-cli/6.0.0.Final/tackle-cli-6.0.0.Final-offline.zip
+RUN microdnf -y install java-11-openjdk-headless openssh-clients unzip wget git subversion maven && microdnf -y clean all
```

Comment:
Please don't flatten this ^ line.  It's much more readable/maintainable on separate lines.